src/settings/App.tsx

```diff
@@ -266,6 +306,15 @@ export class App extends Component<{}, AppState> implements OC.Plugin<OC.Search.
 					onClick={() => {
 						this.setState({editingGroup: 0, editingMountPoint: 0})
 					}}>
+			<div id="groupfolders-admin-delegation">
+				<h3>{ t('groupfolders', 'groupfolders delegation') }</h3>
```

Comment:
```suggestion
				<h3>{ t('groupfolders', 'Group folder admin delegation') }</h3>
```

Should make it more clear that this is about delegating admin settings

Reply:
done